schemas/version/2.0/version.json

```diff
@@ -8,7 +8,10 @@
       "properties": {
         "platform": { "type": "string",  "enum": ["windows", "mac", "android", "ios", "web", "linux"] },
         "level": { "type": "string", "enum": ["stable", "alpha", "beta"] },
-        "version": { "type": "string" }
+        "version": { "type": "string" },
+        "alpha": { "type": "string" },
+        "beta": { "type": "string" },
+        "stable": { "type": "string" }
```

Comment:
Does this need to be in the schema too?

> If a version is not available for a given platform and stability leve, then an `$error-message` is returned instead of `$version`: